pkg/azurefile/controllerserver.go

```diff
@@ -995,7 +995,11 @@ func (d *Driver) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshotRequ
 		itemSnapshotTime = properties.LastModified()
 		itemSnapshotQuota = properties.Quota()
 	} else {
-		snapshotShare, err := d.cloud.FileClient.WithSubscriptionID(subsID).CreateFileShare(ctx, rgName, accountName, &fileclient.ShareOptions{Name: fileShareName, RequestGiB: defaultAzureFileQuota, Metadata: map[string]*string{snapshotNameKey: &snapshotName}}, snapshotsExpand)
```

Comment:
what if just removing `RequestGiB: defaultAzureFileQuota` in L998, will that also fix the issue? that could save on Get file share call

Reply:
No, it default is 0 and will return this error:
```
E1018 01:57:01.111767       1 utils.go:81] GRPC error: rpc error: code = Internal desc = create snapshot from(mc_aks-snapshot_group_aks-snapshot_eastus#f33e8e7dd87ca40fba6f47c#pvc-2e4c617d-86f9-4c7b-a8f4-097633184ca0###default) failed with storage.FileSharesClient#Create: Invalid input: autorest/validation: validation failed: parameter=fileShare.FileShareProperties.ShareQuota constraint=InclusiveMinimum value=0 details: value must be greater than or equal to 1, accountName: "f33e8e7dd87ca40fba6f47c"
```